backends/rollup/rollup-init-transaction/Backend/NFT.hs

```diff
@@ -0,0 +1,40 @@
+{-# LANGUAGE TemplateHaskell #-}
+
+{-# OPTIONS_GHC -fplugin-opt PlutusTx.Plugin:profile-all #-}
+{-# OPTIONS_GHC -fplugin-opt PlutusTx.Plugin:conservative-optimisation #-}
+{-# OPTIONS_GHC -Wno-unrecognised-pragmas #-}
+
+module Backend.NFT where
```

Comment:
I guess we want to move all uplc to `backends`, and bench will be able to import `uplc` from `backends`?

Reply:
Maybe we also want to rename `backends` to `zkfold-backend`?

Reply:
Why do we need this? Because `zkfold-cardano` is the core library, and `UPLC` is really heavy stuff.

Reply:
As a first step I will move content of NFT.hs to appropriate places in UPLC, since it is a (minting) validator.  Then we can decide where we want to place all of UPLC.

Reply:
So I have moved what was on NFT.hs to UPLC.